Makefile

```diff
@@ -145,7 +144,3 @@ packages-sdk: build
 			tar -cvzf $(PROJECT)_$${TAR_VERSION}_$${os}_$${arch}.tar.gz $(PROJECT)_$${os}_$${arch}/; \
 		done; \
 	done
-
-.PHONY: migrate
-migrate: $(DEPENDENCIES)
-	kallax migrate up -d ./store/migrations --dsn '$(DSN)' --all
```

Comment:
Do we still need `DSN` variable in Makefile then?

Reply:
True, I missed that. Fixed in 39cb2a5